src/dmd/expressionsem.d

```diff
@@ -7531,7 +7542,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
 
         if (exp.to.ty == Ttuple)
         {
-            exp.error("cannot cast `%s` to tuple type `%s`", exp.e1.toChars(), exp.to.toChars());
+            exp.error("cannot cast `%s` to tuple type `%s`", exp.e1.type.toChars(), exp.to.toChars());
```

Comment:
The expression should still be printed, it's the main subject of the cast.
Add the type after.

```suggestion
            exp.error("cannot cast `%s` of type `%s` to tuple type `%s`", exp.e1.toChars(), exp.e1.type.toChars(), exp.to.toChars());
```